gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManagerController.java

```diff
@@ -87,4 +94,24 @@ private ResponseEntity<byte[]> getZippedGroup(OffsetDateTime timestamp, String f
     public ResponseEntity<List<TaskDto>> getListTasksFromBusinessDate(@PathVariable String businessDate) {
         return ResponseEntity.ok().body(builder.getListTasksDto(LocalDate.parse(businessDate)));
     }
+
+    @GetMapping(value = "/tasks/{timestamp}/file/{fileType}", produces = "application/octet-stream")
+    public @ResponseBody ResponseEntity<byte[]> getFile(@PathVariable String fileType, @PathVariable String timestamp) throws IOException {
+        ResponseEntity<byte[]> result = ResponseEntity.notFound().build();
```

Comment:
I would rather not define this result variable. And directly return the response entities in both cases. It would increase readability